pallets/gear/src/internal.rs

```diff
@@ -488,6 +506,7 @@ where
             mailboxed.id(),
             hold_interval,
             CostsPerBlockOf::<T>::mailbox(),
+            None,
```

Comment:
won't there be similar max amount? ref to comment on charge for hold fn

Reply:
Yes. The reason (as explained before) is that we haven't locked anything for `Cut` nodes because the only possible use case for this type of node is to pay mailbox rent. For the sake of uniformity will add locking for this node type, as well.